### ISSUE TITLE
Crimea is part of Ukraine, not Russia!

### ISSUE THREAD

**Issue by FabriceN** [nvkelso/natural-earth-vector#844]
I believe it's arc 155.
Please fix this 'mistake' (no wonder where this error comes from…)

**Comment by RonaldCalitri** [comment]
Actually, Crimea is so polyglot that it has special status in the Ukranian political order. 
But, 'originally' for a couple of millenia, it was Greek - influenced. No wonder the icy rusi hold on to it like a hot potato.

**Comment by megamster** [comment]
@FabriceN Natural Earth attempts to reflect de facto rather than de jure boundaries. As such, Crimea is at best a disputed territory, not Ukrainian. Honestly, sticking to de facto status of any given territory seems to be the only sane policy to have. There are hundreds of territories that are disputed by two or more countries around the world

**Comment by FabriceN** [comment]
Well… this is precisely the reason why we have the United Nations and the international recognition of borders.
Crimea is, without a doubt, Ukranian.
Applying your policy would mean taking out Donbass, but also changing the borders of Morocco, Sudan, Dem. Rep of Congo…
And by the way, you should add animations to the svg file, since the de facto changes every day.

No, the truth is more straight forward: you found this open source file and insidiously adapted to conform to Putler’s view.
That puts you very far from a position where the word “honestly” makes any sense in your sentence.


De : megamster ***@***.***>
Date : lundi, 22 mai 2023 à 11:04
À : nvkelso/natural-earth-vector ***@***.***>
Cc : Fabrice Nordmann ***@***.***>, Mention ***@***.***>
Objet : Re: [nvkelso/natural-earth-vector] Crimea is part of Ukraine, not Russia! (Issue #844)

@FabriceN<https://github.com/FabriceN> Natural Earth attempts to reflect de facto rather than de jure boundaries. As such, Crimea is at best a disputed territory, not Ukrainian. Honestly, sticking to de facto status of any given territory seems to the only sane policy to have. There are hundreds of territories that are disputed by two or more countries around the world

**Comment by megamster** [comment]
@FabriceN the vast majority of maps stick to de facto as opposed to de jure boundaries, its not something exclusive to this project and in most developed countries kids learn this simple fact at school as they're learning to interpret maps. Maybes the borders you mention need redrawing and I'm sure your contributions to that would be helpful to the project. 

Having said that, to go to a de jure policy would mean the redrawing of several other borders, like the one between continental Portugal and Spain. Olivença is internationally recognized by literally everyone except Spain, including the UN, as Portuguese. Hell, even Spain itself recognized it as such at several points, it just never bothered to handover the place. Yet, basically every map puts it as Spanish. Why? Because its de facto held by Spain and intelligent people understand that.

As for borders that are hanging hands frequently, the disputed labelling does exist.

As for your comment about the United Nations, that's the last authority one would go for a usable map. The reason is obvious. The UN provides a de jure or, in other words, phantasy view of the world.

Want a straightforward truth?
- Somehow, you only seem to care about a case you seem to be emotionally invested on and not in having a coherent policy that makes sense.... You want to change the longstanding standard policy for such cases based on your feelings towards a particular situation. 

Dishonesty or embarrassing ignorance, pick your poison.

Also, note that I didn't touch or adapt any file

**Comment by FabriceN** [comment]
Texas should be México then.
________________________________
De : megamster ***@***.***>
Envoyé : Monday, May 22, 2023 5:59:19 PM
À : nvkelso/natural-earth-vector ***@***.***>
Cc : Fabrice Nordmann ***@***.***>; Mention ***@***.***>
Objet : Re: [nvkelso/natural-earth-vector] Crimea is part of Ukraine, not Russia! (Issue #844)


@FabriceN<https://github.com/FabriceN> the vast majority of maps stick to de facto as opposed to de jure boundaries, its not something exclusive to this project and in most developed countries kids learn this simple fact at school as they're learning to interpret maps. Maybes the borders you mention need redrawing and I'm sure your contributions to that would be helpful to the project.

Having said that, to go to a de jure policy would mean the redrawing of several other borders, like the one between continental Portugal and Spain. Olivença is internationally recognized by literally everyone except Spain, including the UN, as Portuguese. Hell, even Spain itself recognized it as such at several points, it just never bothered to handover the place.

As for borders that are hanging hands frequently, the disputed labelling does exist.

As for your comment about the United Nations, that's the last authority one would go for a usable map. The reason is obvious. The UN provides a de jure or, in other words, phantasy view of the world.

What a straightforward truth?

  *   Somehow, you only seem to care about a case you seem to be emotionally invested on and not in having a coherent policy that makes sense.... You want to change the longstanding standard policy for such cases based on your feelings towards a particular situation.

Dishonesty or embarrassing ignorance, pick your poison.

Also, not that I didn't touch or adapt any file

**Comment by megamster** [comment]
@FabriceN yeah, according to your silly proposal to use UN maps and not actual on the ground control maybe it would. Just shows how nonsensical your arguments are if you think about them

**Comment by FabriceN** [comment]
Hurry, there are some news from Belgorod oblast. You should update your de facto map quickly
________________________________
De : megamster ***@***.***>
Envoyé : Monday, May 22, 2023 6:10:47 PM
À : nvkelso/natural-earth-vector ***@***.***>
Cc : Fabrice Nordmann ***@***.***>; Mention ***@***.***>
Objet : Re: [nvkelso/natural-earth-vector] Crimea is part of Ukraine, not Russia! (Issue #844)


@FabriceN<https://github.com/FabriceN> yeah, according to your silly proposal to use UN maps and not actual on the ground control maybe it would. Just shows how nonsensical your arguments are

**Comment by megamster** [comment]
@FabriceN its as much mine as it is yours. Any TV news channel marks the region as disputed. Natural Earth has that same ability so that you don't need to update it every second. Kids at school also do learn about disputed areas on a map. I only say this so that you understand how silly you come off as with such comments

**Comment by nvkelso** [comment]
The discussion here has veered into uncharted territories, as it were. I'm locking comments on this issue for 6 months and will start blocking GitHub users who can't maintain decorum.
